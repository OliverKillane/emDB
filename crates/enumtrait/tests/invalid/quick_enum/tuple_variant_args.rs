#[enumtrait::quick_enum]
enum Invalid1 {
    Bing(i32, i32)
}

fn main() {}