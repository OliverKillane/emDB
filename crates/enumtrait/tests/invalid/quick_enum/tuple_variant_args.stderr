error: Provided variants should be a single identifier of the type.

         = help: Try `enum Invalid1 { .. Bing(<type>), .. }`

 --> tests/invalid/quick_enum/tuple_variant_args.rs:3:10
  |
3 |     Bing(i32, i32)
  |          ^^^^^^^^
